### PR TITLE
fix flicker

### LINKED ISSUE #45: flickering display
Running tmux git and fully upgraded tmux-thumbs, behold the flickering in alacritty :)
Would it be possible for tmux-thumbs to not constantly re-draw the "overlay"?
flickering

### PR DESCRIPTION
**Pull request by pbartyik**
to fix #45 , this way it only flickers when moving the selection with arrows

### CODE REVIEW

File: src/view.rs [hunk 1]
```diff
@@ -242,6 +242,7 @@ impl<'a> View<'a> {
         _ => {
           // Nothing in the buffer. Wait for a bit...
           std::thread::sleep(std::time::Duration::from_millis(100));
+          continue; // don't render again if nothing new to show
```
Comment (fcsonline) [inline src/view.rs:245]: Nice! Also, I'm going to tweak a bit another case that flickers. When you have many keystrokes in the stdin (pushing long keys), we are saying to `termion` to print many things. I'm going to throttle this.

Comment (fcsonline) [reply src/view.rs:245]: https://github.com/fcsonline/tmux-thumbs/commit/10eff07d160bd53c0949fcb66e623687ba01b668

**Comment by fcsonline** [comment]
Thanks for your collaboration! :clap:

**Comment by pbartyik** [comment]
First open source contribution :)

**Comment by fcsonline** [comment]
An awesome one! :wink:

**Pull request merged**